src/static/menu-win.json

```
@@ -81,6 +81,15 @@
                     "separator": true
                 },
                 {
+                    "id": "EXPORT_UTILITY",
+                    "shortcut": {
+                        "keyChar": true,
+                        "modifiers": {
+                            "command": true
```

`control`. Also, the menu infrastructure should really throw an error here!